### PR TITLE
[Feature][connector-v2][PostgreSQL CDC]Support source PostgreSQL CDC

### PR DESCRIPTION
**Pull request by ediconss**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by zhilinli123** [comment]
Thank you for your contribution, you can try to merge the latest branch, and the CDC to v2 version of the document changes Specific reference: https://seatunnel.apache.org/docs/2.3.3/connector-v2/source/MySQL-CDC
https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/MySQL-CDC.md

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-postgres-e2e/src/test/java/org/apache/seatunnel/e2e/connector/cdc/postgres/PostgresCDCIT.java [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    @Override
+    public void startUp() throws ClassNotFoundException, InterruptedException {
+        log.info("The second stage: Starting postgres containers...");
+        Startables.deepStart(Stream.of(POSTGRES_CONTAINER)).join();
+        log.info("postgres Containers are started");
+        log.info("postgres ddl execution is complete");
+    }
+
+    @TestTemplate
+    public void testpostgresCdcCheckDataE2e(TestContainer container)
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-postgres-e2e/src/test/java/org/apache/seatunnel/e2e/connector/cdc/postgres/PostgresCDCIT.java]: Please refer to this  testcase

https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java#L177

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-postgres-e2e/src/test/java/org/apache/seatunnel/e2e/connector/cdc/postgres/PostgresCDCIT.java [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.awaitility.Awaitility.await;
+
+@Slf4j
+@DisabledOnContainer(
+        value = {},
+        type = {EngineType.SPARK, EngineType.FLINK},
+        disabledReason = "Currently SPARK and FLINK do not support cdc")
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-postgres-e2e/src/test/java/org/apache/seatunnel/e2e/connector/cdc/postgres/PostgresCDCIT.java]: remove？
```suggestion
        type = {EngineType.SPARK},
        disabledReason = "Currently SPARK do not support cdc")
```

Comment (ediconss) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-postgres-e2e/src/test/java/org/apache/seatunnel/e2e/connector/cdc/postgres/PostgresCDCIT.java]: > remove？

done

**Comment by ediconss** [comment]
> Thank you for your contribution, you can try to merge the latest branch, and the CDC to v2 version of the document changes Specific reference: https://seatunnel.apache.org/docs/2.3.3/connector-v2/source/MySQL-CDC https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/MySQL-CDC.md

done

**Comment by ediconss** [comment]
> Thank you for your contribution, you can try to merge the latest branch, and the CDC to v2 version of the document changes Specific reference: https://seatunnel.apache.org/docs/2.3.3/connector-v2/source/MySQL-CDC https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/source/MySQL-CDC.md

done

**Pull request closed without merging**